js/script.js

```diff
@@ -11,83 +11,83 @@
+  li.setAttribute('contentEditable', 'false');
 
-	//Create a checkbox
-	var checkbox = document.createElement("INPUT");
-	checkbox.setAttribute("type", "checkbox");
-	// checkbox.setAttribute("onclick", "toggleChecked();") //no longer needed
-	checkbox.className += ' listcheckbox';
-	li.appendChild(checkbox);
+  //Create a checkbox
+  var checkbox = document.createElement("INPUT");
+  checkbox.setAttribute("type", "checkbox");
+  // checkbox.setAttribute("onclick", "toggleChecked();") //no longer needed
+  checkbox.className += ' listcheckbox';
+  li.appendChild(checkbox);
 
-	//Fetch the user input data
-	var inputItem = document.getElementById("textbox").value;
+  //Fetch the user input data
+  var inputItem = document.getElementById("textbox").value;
 
-	//Create a text node with the input data
-	// var newNode = document.createTextNode(inputItem);
-	// li.appendChild(newNode);
+  //Create a text node with the input data
+  // var newNode = document.createTextNode(inputItem);
+  // li.appendChild(newNode);
 
-	//Check if there is an input
-	if(inputItem === ''){
-		alert("Please enter a to-do task!");
-	}
+  //Check if there is an input
+  if (inputItem === '') {
+    alert("Please enter a to-do task!");
+  }
 
-	else{
-		document.getElementById("list").appendChild(li);
-	}
+  else {
+    document.getElementById("list").appendChild(li);
+  }
 
-	//TRY LABEL
-	var span3 = document.createElement("LABEL");
-	var txt3 = document.createTextNode(inputItem);
-	span3.appendChild(txt3);
-	span3.className = "listdata";
-	li.appendChild(span3);
-	//TRY LABEL
+  //TRY LABEL
+  var span3 = document.createElement("LABEL");
+  var txt3 = document.createTextNode(inputItem);
+  span3.appendChild(txt3);
+  span3.className = "listdata";
+  li.appendChild(span3);
+  //TRY LABEL
 
-	//TRY THIS - Contain in SPAN
-	// var span2 = document.createElement("SPAN");
-	// var txt2 = document.createTextNode(inputItem);
-	// span2.appendChild(txt2);
-	// span2.className = "listdata";
-	// li.appendChild(span2);
-	//TRY THIS
+  //TRY THIS - Contain in SPAN
+  // var span2 = document.createElement("SPAN");
+  // var txt2 = document.createTextNode(inputItem);
+  // span2.appendChild(txt2);
+  // span2.className = "listdata";
+  // li.appendChild(span2);
+  //TRY THIS
 
-	document.getElementById("textbox").value = "";
+  document.getElementById("textbox").value = "";
 
-	var span = document.createElement("SPAN");
+  var span = document.createElement("SPAN");
 
-	var txt = document.createTextNode(" \u00D7");
+  var txt = document.createTextNode(" \u00D7");
 
-	span.className = "close";
+  span.className = "close";
 
-	span.appendChild(txt);
+  span.appendChild(txt);
 
-	li.appendChild(span);
+  li.appendChild(span);
 
-	for(i = 0; i < close.length; i++){
-		close[i].onclick = function(){
-			var div = this.parentElement;
-			div.style.display = "none";
-		}
-	}
+  for (i = 0; i < close.length; i++) {
+    close[i].onclick = function () {
+      var div = this.parentElement;
```

Comment:
use better variable names. If I'm a random person reading this, I won't know what `div` is